db/schema.rb

```diff
@@ -347,19 +347,19 @@
 
   create_table "profiles", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "person_id", null: false
-    t.string "last_name", null: false
-    t.string "first_names", null: false
-    t.date "date_of_birth"
-    t.string "aliases", default: [], array: true
-    t.uuid "gender_id"
-    t.uuid "ethnicity_id"
-    t.uuid "nationality_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.jsonb "assessment_answers"
-    t.jsonb "profile_identifiers"
-    t.string "gender_additional_information"
     t.integer "latest_nomis_booking_id"
+    t.string "gender_additional_information"
+    t.jsonb "profile_identifiers"
+    t.uuid "nationality_id"
+    t.uuid "ethnicity_id"
+    t.uuid "gender_id"
+    t.string "aliases"
```

Comment:
this was an array `t.string "aliases", default: [], array: true` but now it isn't? has something else changed?